external-reports/src/test/java/org/sonar/java/externalreport/PmdSensorTest.java

```diff
@@ -62,6 +62,7 @@ void test_descriptor() {
     DefaultSensorDescriptor sensorDescriptor = new DefaultSensorDescriptor();
     sensor.describe(sensorDescriptor);
     assertThat(sensorDescriptor.name()).isEqualTo("Import of PMD issues");
+    assertThat(sensorDescriptor.languages()).isEqualTo(List.of("java"));
```

Comment:
Because we set the only value to be java, but we don't know what kind of data structure holds the information, it might be safer to remove the list type from the test
```suggestion
    assertThat(sensorDescriptor.languages()).containsOnly("java");
```